common/api-review/auth.api.md

```diff
@@ -564,7 +564,7 @@ export interface ParsedToken {
 
 // @public
 export interface PasswordPolicy {
-    readonly allowedNonAlphanumericCharacters: string[];
+    readonly allowedNonAlphanumericCharacters?: string[];
```

Comment:
I wonder why? and this should align with our api proposal

Reply:
I tested this with the default policy returned from the backend and this field was undefined (even though it was not marked as optional in the proto). To keep this aligned with the API proposal, I'll revert this change and have this field be an empty array if it is not defined in the backend response.

Reply:
I'll also convert this to be a `string` instead of `string[]` in a future PR, since that is part of the API proposal as well.